build.gradle

```diff
@@ -155,6 +155,7 @@ buildScan {
  */
 
 ext.minSdkVersion = 14
+ext.supportLibraryVersion = '25.4.0'
```

Comment:
please use the block form:
ext {
    minSdkVersion = 14
    supportLibraryVersion = '25.4.0'
}